mama/c_cpp/src/c/mama/publisher.h

```diff
@@ -154,8 +232,8 @@ mamaPublisher_sendFromInbox (mamaPublisher publisher,
 MAMAExpDLL
 extern mama_status
 mamaPublisher_sendReplyToInboxHandle (mamaPublisher publisher,
-                                	  mamaMsgReply replyAddress,
-                                	  mamaMsg reply);
+                                      mamaMsgReply replyAddress,
```

Comment:
It's the argument list I'm talking about - the spaces are fine, so:

```
mamaPublisher_sendReplyToInboxHandle (mamaPublisher publisher,
                                	  mamaMsgReply  replyAddress,
                                      mamaMsgReply  replyAddress,
                                      mamaMsg       reply);
```
Rather than:
```
mamaPublisher_sendReplyToInboxHandle (mamaPublisher publisher,
                                	  mamaMsgReply replyAddress,
                                      mamaMsgReply replyAddress,
                                      mamaMsg reply);
```